Assets/Treasure/Example/Scripts/IdentityUI.cs

```diff
@@ -1,4 +1,6 @@
 using System;
```

Comment:
Should we rename this file to SessionUI?

Reply:
Just going to keep it as-is for now to minimize file and scene changes 🫡